### DISCUSSION TITLE
On Windows OS: Installing cookbook > LLMS > GORQ > Video Summary

### DISCUSSION THREAD

**Discussion by tsktech** [agno-agi/agno#208] [Show and tell]
**STEP 1**
```
python3 -m venv ~/.venvs/aienv
source ~/.venvs/aienv/bin/activate
```

is has flows (btw i run anaconda on win11)
**_for windows_**
```
(base) XXXX> conda env list
check if the environment already existing
(base) XXXX> conda create -n aienv python=3.12
(base) XXXX> conda activate aienv
```

**STEP 2** _Note the base is changed to aienv_
`(aienv) XXXX> export GROQ_API_KEY=***`

for windows export is set
`(aienv) XXXX> set GROQ_API_KEY=***`

**STEP 3**
`pip install -r cookbook/llms/groq/video_summary/requirements.txt`

**the best solution was to git clone https://github.com/phidatahq/phidata.git or you could download the repo.zip file and unzip it to a folder**
mine was unzipped to C:\User\XXXX\  (WIN does not have git installed by default)
`(aienv) XXXX> pip install -r  C:\User\XXXX\Phidata\cookbook\llms\groq\video_summary\requirements.txt`

**STEP 4**
`streamlit run cookbook/llms/groq/video_summary/app.py`
on windows
`(aienv) XXXX> streamlit run C:\User\XXXX\Phidata\cookbook\llms\groq\video_summary\app.py`
**_I got an error_** 
`'streamlit' is not recognized as an internal or external command, operable program or batch file.`

I did the following to add channel as detailed [conda-forge.org](https://conda-forge.org/docs/user/introduction/)
```
(aienv) XXXX> conda config --add channels conda-forge
(aienv) XXXX> conda install streamlit
```
after its installed try
`(aienv) XXXX> streamlit run C:\User\XXXX\Phidata\cookbook\llms\groq\video_summary\app.py`

Regards
Srikanth Kamath

{ Note:
if you get user permission error, then run-in privileged mode the "anaconda Prompt" --> "Right click run as administrator" 
```
(base) XXXX> conda activate aienv
(aienv) XXXX> conda config --add channels conda-forge
(aienv) XXXX> conda install streamlit
```
}

![2024-05-02 09_35_52-Anaconda Powershell Prompt](https://github.com/phidatahq/phidata/assets/49195633/fcdcba95-639d-4d92-aee1-f94b21c8920d)

### **and there we go !! thanks @ashpreetbedi @ysolanky and all other contributors**

![2024-05-02 09_36_55-Youtube Video Summaries and 12 more pages - Personal - Microsoft​ Edge](https://github.com/phidatahq/phidata/assets/49195633/111c5e82-8106-435b-9bd4-ab6aabdccc77)

**Comment by ashpreetbedi** [comment]
<3 love this @tsktech
